src/EditorFeatures/Core/Wrapping/BinaryExpression/AbstractBinaryExpressionWrapper.cs

```diff
@@ -17,7 +16,7 @@ internal abstract partial class AbstractBinaryExpressionWrapper<TBinaryExpressio
         private readonly IPrecedenceService _precedenceService;
 
         protected AbstractBinaryExpressionWrapper(
-            IBlankLineIndentationService indentationService,
+            Indentation.IIndentationService indentationService,
```

Comment:
>Indentation [](start = 12, length = 11)

Let's add a using instead of qualifying.

Reply:
sure.

Reply:
Done.